src/profile-logic/import/chrome.js

```diff
@@ -204,13 +204,14 @@ function findEvent<T: TracingEventUnion>(
 }
 
 function getThreadInfo<T: Object>(
-  threadInfoByTid: Map<number, ThreadInfo>,
+  threadInfoByPidAndTid: Map<string, ThreadInfo>,
   threadInfoByThread: Map<Thread, ThreadInfo>,
   eventsByName: Map<string, TracingEventUnion[]>,
   profile: Profile,
   chunk: TracingEvent<T>
 ): ThreadInfo {
-  const cachedThreadInfo = threadInfoByTid.get(chunk.tid);
+  const pidAndTid = `${chunk.pid}:${chunk.tid}`;
```

Comment:
Optional: The commit message may be nice as a comment so it's more persistent

>  Handle duplicate tids with different pids.
> I have a profile where the main threads of two different processes both have tid 775 for some reason.
> This caused the markers from a renderer process to be mistakenly assigned to the GPU process.

Reply:
added a comment